src/tools/render_passes/mod.rs

```diff
@@ -188,13 +189,36 @@ impl RenderPass for HideAreas {
 }
 
 #[derive(Default)]
-pub struct HideInvisible;
+pub struct HideInvisible {
+    overrides: Vec<String>,
+}
+
 impl RenderPass for HideInvisible {
+    fn configure(&mut self, renderer_config: &dm::config::MapRenderer) {
+        self.overrides = renderer_config.force_hide_types.clone().into_iter().collect::<Vec<_>>();
+        // Put longer typepaths earlier in the list so that `/foo/bar` can override `/foo`.
+        self.overrides.sort_unstable_by_key(|k| usize::MAX - k.len());
```

Comment:
Dont know if this is needed or not given that this list doesnt have overrides. Its just a path list.